src/components/Contributors.tsx

```diff
@@ -64,7 +64,9 @@ const Contributors = () => {
           <Card className="border-none -m-5">
             <CardHeader>
               <p>Chords:{VERSION}</p>
-              <CardTitle className="text-lg">Contributors</CardTitle>
+              <CardTitle className="text-lg">
+                Contributors (in alphabetical order)
+              </CardTitle>
```

Comment:
Will change UI so that title don't contain (in alphabetical order) but it will be mentioned.

Reply:
Add this div inside card title, and remove text-lg from card title
<div className="flex justify-between">
<span className="text-lg">Contributors</span>
<span className="text-sm text-muted-foreground">Alphabetical Order</span>
</div>